.github/workflows/integration-test.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Install Dependencies
         shell: bash
         run: |
-          sudo apt update && sudo apt install -y --no-install-recommends build-essential make libncurses5
+          sudo apt update && sudo apt install -y --no-install-recommends build-essential make
```

Comment:
Oh no, same here with regarding the tests failing?

Reply:
Yep, presumably because v1.14/v1.15 still use llvm 10 and the binary we use is linked against `libtinfo.so.5`. Not sure what the best way forward is? Drop the backport and add a v1.14/v1.15 only fix that pins the runner image at `ubuntu-22.04` maybe?

Reply:
I've dropped #35408 from this PR for now and marked it as `backport/author` so we can figure out a solution, also see https://github.com/cilium/cilium/pull/35408#issuecomment-2431288124